swarm/storage/hasherstore.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2016 The go-ethereum Authors
```

Comment:
Bump up copyright to 2018.

Reply:
@nonsense Fixed in 46e98327d70708b29706cbc7f87e975306b0c2a1